### DISCUSSION TITLE
Steam support?

### DISCUSSION THREAD

**Discussion by Bibue35** [MythicApp/Mythic#138] [Q&A]
Hey, I was just wondering when Mythic will have Steam support. 
I know It says coming soon, but it's been a while since it says that.
Do you have an approximate time stamp, or like if it would be this or next year?

**Comment by zenfty** [comment]
you can monitor progress here for now:
https://github.com/MythicApp/Mythic/tree/steam
https://github.com/orgs/MythicApp/projects/4/views/1
